src/Parser/Druid/Feral/Modules/Bleeds/RakeSnapshot.js

```diff
@@ -4,23 +4,29 @@ import SpellLink from 'common/SpellLink';
 import { formatPercentage } from 'common/format';
 import Snapshot, { JAGGED_WOUNDS_MODIFIER, PANDEMIC_FRACTION } from '../FeralCore/Snapshot';
 
-/*
-Identify inefficient refreshes of the Rake DoT:
-  Early refresh of a Rake doing double damage due to Prowl with one that will not do double damage.
-  Pre-pandemic refresh of a Rake doing bonus damage from snapshot buffs with one that will do less damage.
-*/
+/**
+ * Identify inefficient refreshes of the Rake DoT:
+ *  Early refresh of a Rake doing double damage due to Prowl with one that will not do double damage.
+ *  Pre-pandemic refresh of a Rake doing bonus damage from snapshot buffs with one that will do less damage.
+ */
 
-// When you cannot refresh a prowl-buffed rake with prowl, ideally you'd let it tick down. At the
-// moment that it expires apply your new one.
-// Looking for exact timing is unrealistic, so give some leeway.
+/**
+ * When you cannot refresh a prowl-buffed rake with prowl, ideally you'd let it tick down.
+ * Then at the moment that it expires you'd apply a fresh DoT.
+ * But exact timing is unrealistic, so give some leeway.
+ */
 const FORGIVE_PROWL_LOSS_TIME = 1000;
 
 const RAKE_BASE_DURATION = 15000;
 class RakeSnapshot extends Snapshot {
-  rakeCastCount = 0;        // count of rake casts of all kinds
-  prowlLostCastCount = 0;   // count of rake buffed with Prowl ending early due to refresh without Prowl buff
-  prowlLostTimeSum = 0;     // total time cut out from the end of prowl-buffed rake bleeds by refreshing early (milliseconds)
-  downgradeCastCount = 0;   // count of rake DoTs refreshed with weaker snapshot before pandemic
+  // rake buffed with Prowl ending early due to refresh without Prowl buff
+  prowlLostCastCount = 0;
+
+  // total time cut out from the end of prowl-buffed rake bleeds by refreshing early (milliseconds)
+  prowlLostTimeSum = 0;
+
+  // rake DoTs refreshed with weaker snapshot before pandemic
+  downgradeCastCount = 0;
 
   on_initialized() {
     this.spellCastId = SPELLS.RAKE.id;
```

Comment:
I think you should set this as a class property instead. Also since it never changes, how about making it static instead?